inference-engine/src/transformations/src/ngraph_ops/convolution_ie.cpp

```diff
@@ -59,23 +104,12 @@ void op::ConvolutionIE::validate_and_infer_types() {
     PartialShape filters_shape = get_input_partial_shape(1);
     element::Type filters_et = get_input_element_type(1);
 
-    element::Type result_et;
-
-    NODE_VALIDATION_CHECK(
-        this,
-        element::Type::merge(result_et, data_batch_et, filters_et),
-        "Element types for data batch and filters do not match (data batch element type: ",
-        data_batch_et,
-        ", filters element type: ",
-        filters_et,
-        ").");
-
```

Comment:
You just remove requirements of node to match precisions on activation and weight. It change semantic of this node. 
@GlebKazantaev is it legal allowance?